src/index.html

```diff
@@ -9,5 +9,48 @@
   </head>
   <body>
     <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
It makes sense for these stars to be an `ul` with `li`'s rather than just divs, this block will be reused in your future prs so it will future-proof it. 
Generally it's a bad practice to use divs for everything, avoid it as much as possible.